_posts/2019-11-25-Moving-to-Gitter.md

```diff
@@ -0,0 +1,38 @@
+---
+author: Polybar Team
+account: polybar
+title: Moving the Polybar Community to Gitter
+---
+
+Right now the polybar community lives mainly on IRC and Reddit (and Github of 
```

Comment:
> the Polybar community

or even

> The Polybar Community

perhaps?

Also maybe it would make sense to turn `IRC`, `Reddit` and `Github` into links here?